docs/stylesheets/additional_coloring.css

```diff
@@ -0,0 +1,58 @@
+[data-md-color-primary="teal"] {
+    --async-color: 255, 92, 122;
+    --property-color: 38, 166, 153;
+    --classmethod-color: 0, 198, 245;
+    --class-attribute-color: 230, 170, 0;
+    --instance-attribute-color: 255, 61, 220;
```

Comment:
```suggestion
    --instance-attribute-color: 181, 156, 255;
```

How about a color that is a little easier on the eyes for this one? Looks something like this:

![Screenshot_20240504_215107](https://github.com/hypergonial/hikari-arc/assets/46067571/e803356a-1b81-4270-a199-5c531545bdd5)

Reply:
No problemo